std/utf.d

```diff
@@ -1198,7 +1198,7 @@ dchar decodeFront(UseReplacementDchar useReplacementDchar = UseReplacementDchar.
 }
 
 // Gives the maximum value that a code unit for the given range type can hold.
-private template codeUnitLimit(S)
+package template codeUnitLimit(S)
```

Comment:
`codeUnitLimit` is selectively imported in `std.string`, so need to have `package` access at least.